solidity/test/rewards/TestECDSARewards.js

```diff
@@ -202,14 +206,45 @@ describe("ECDSARewards", () => {
 
       // verify second keep in this interval
       keepAddress = await keepFactory.getKeepAtIndex(1)
+      keepAddresses = [keepAddress]
       keep = await BondedECDSAKeepStub.at(keepAddress)
       await keep.publicMarkAsClosed()
 
-      await rewardsContract.receiveReward(keepAddress)
+      await rewardsContract.receiveReward(keepAddresses)
 
       // 297,000 * 2 = 594,000
       await assertKeepBalanceOfBeneficiaries(expectedBeneficiaryBalance.muln(2))
     })
+
+    it("should correctly receive rewards from multiple keeps", async () => {
+      for (let i = 0; i < 10; i++) {
+        await keepFactory.stubOpenKeep(owner, operators, firstIntervalStart)
+      }
+
+      await timeJumpToEndOfInterval(0)
+
+      const keepAddress = await keepFactory.getKeepAtIndex(0)
+      const keepAddress1 = await keepFactory.getKeepAtIndex(1)
+      const keepAddress2 = await keepFactory.getKeepAtIndex(2)
+      const keepAddresses = [keepAddress, keepAddress1, keepAddress2]
+      const keep = await BondedECDSAKeepStub.at(keepAddress)
+      const keep1 = await BondedECDSAKeepStub.at(keepAddress1)
+      const keep2 = await BondedECDSAKeepStub.at(keepAddress2)
+
+      await keep.publicMarkAsClosed()
+      await keep1.publicMarkAsClosed()
+      await keep2.publicMarkAsClosed()
+
+      await rewardsContract.receiveReward(keepAddresses)
+
+      // reward for the first interval: 7,128,000 KEEP
+      // keeps created: 10 => 712,800 KEEP per keep
+      // member receives: 712,800 / 3 = 237,600 (3 signers per keep)
+      // member was in 3 properly closed keeps: 237,600 * 3 = 712,800
```

Comment:
Can we do 2 or 4 properly closed keeps? Just to have a different number than 712,800 that is the same as an allocation for the whole keep.

Reply:
We can, changed to 2 closed keeps.